### PR TITLE
added convert to F

### LINKED ISSUE #3: Developing convertToF() function
TO DO:
- [x] Develop function ```convertToF()```
- [x] Test its behaviour
- [x] Add to the interface

### PR DESCRIPTION
**Pull request by gelilaa**
Closes #3

### CODE REVIEW

File: scripts/convert-to-f.js [hunk 1]
```diff
@@ -46,11 +49,27 @@ function convertToFHandler() {
...
+  
 
   // core logic
-  const result = convertToF(c);
-
+  
+  const result =convertToF(celsius) ;
+  
   // display for use
+  var message=( celsius + '\xB0  is ' + result + ' \xB0 .');
```
Comment (pavelbidenko2018) [inline scripts/convert-to-f.js]: Could I pls propose to change the output a little: 
var message=( celsius + '\xB0 **C** is ' + result + ' \xB0 **F** .');

**Review by pavelbidenko2018 — CHANGES_REQUESTED** [review]
Nicely done!! 
If we should include the bunch of tests like @fmkarakus ?

File: scripts/convert-to-f.js (hunk 1 above)
Comment (gelilaa) [reply scripts/convert-to-f.js]: that is good. let me try to change it.

Comment (gelilaa) [reply scripts/convert-to-f.js]: It is done.

**Review by pavelbidenko2018 — APPROVED** [review]
thnx )

**Review by fmkarakus — COMMENTED** [review]
Everything looks fine to me. Nicely done!

**Pull request merged**